_posts/2023-03-08-quarkus-3-0-0-alpha5-released.adoc

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title: 'Quarkus 3.0.0.Alpha5 released - now with Hibernate 6, new Developer Console and more!'
+date: 2023-02-18
+tags: release
+synopsis: 'This version is the second biggest update with major impact for Quarkus 3 - upgrading Hibernate 5 to Hibernate 6.'
+author: maxandersen
+---
+
+We are in the process of getting Quarkus 3 ready - last we got our main branch to Jakarta EE 10 to let us getting more things integrated and tested faster.
+
+The big one this time is the move from Hibernate 5 to Hibernate 6 - which *will* cause breakages thus please read the section below for details if you are Hibernate ORM or Hibernate Reactive user.
+
+  * https://github.com/quarkusio/quarkus/pull/31235[Hibernate ORM 6] & Hibernate Reactive 2
+  * @Inject for https://github.com/quarkusio/quarkus/pull/31392[Hibernate StatelessSession]
+  * https://github.com/quarkusio/quarkus/pull/31175[Multiple Persistence Unit] support for Spring Data repositories
+  * New Developer Console
+  * `quarkus deploy` in Quarkus CLI
+  * `@Scheduled` https://github.com/quarkusio/quarkus/pull/31343[support for timezones]
+  * Support https://github.com/quarkusio/quarkus/pull/31192[HTTP/2 connections] in REST Client Reactive
+  * `@ClientRedirectHandler` https://github.com/quarkusio/quarkus/pull/31142[to support custom redirect in REST Client Reactive]
+  * Add support for https://github.com/quarkusio/quarkus/pull/31159[Gradle 8]
+  * https://github.com/quarkusio/quarkus/pull/31079[Multiple OIDC code flows now allowed in same browser]
+    
+  
+== Hibernate ORM 6
+
+Hibernate ORM 6 is a major upgrade to the main persistence layer of Quarkus. It brings a lot of improvements and new features, but also some breaking changes. We have done our best to make the migration as smooth as possible, but some changes are unavoidable. 
+
+We've written https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.0:-Hibernate-ORM-5-to-6-migration[Hibernate ORM 5 to 6 migration] guide for Quarkus users to help you with the migration.
+
+Take particular note of the https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.0:-Hibernate-ORM-5-to-6-migration#database-orm-compatibility["Best-effort Hibernate ORM 5.6 compatability switch"] which reduces the impact of the default schema changes in Hibernate 6.
```

Comment:
Maybe let's be clear that this switch is not intended for long-term use?

```suggestion
Take particular note of the https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.0:-Hibernate-ORM-5-to-6-migration#database-orm-compatibility["Best-effort Hibernate ORM 5.6 compatability switch"] which can temporarilly reduce the impact of the default schema changes in Hibernate 6 and give you more time to migrate.
```

Reply:
slightly reduced it t not make it a too complex sentence to parse :)